tools/fragpipe/fragpipe.xml

```diff
@@ -0,0 +1,276 @@
+            <discover_datasets pattern="__designation__" ext="tabular" directory="outputs/combined_outputs"/>
+        </collection>
+
+        <!-- Galaxy-wrapper concatenated results -->
+        <data name="concat_psm_tsv" format="tabular" label="${tool.name} on ${on_string}: Concatenated psm.tsv" from_work_dir="outputs/concat_psm.tsv">
+            <filter>not workflow_only and 'concatenated_outputs' in output_options</filter>
+        </data>
+        <data name="concat_ion_tsv" format="tabular" label="${tool.name} on ${on_string}: Concatenated ion.tsv" from_work_dir="outputs/concat_ion.tsv">
+            <filter>not workflow_only and 'concatenated_outputs' in output_options</filter>
+        </data>
+        <data name="concat_peptide_tsv" format="tabular" label="${tool.name} on ${on_string}: Concatenated peptide.tsv" from_work_dir="outputs/concat_peptide.tsv">
+            <filter>not workflow_only and 'concatenated_outputs' in output_options</filter>
+        </data>
+        <data name="concat_protein_tsv" format="tabular" label="${tool.name} on ${on_string}: Concatenated protein.tsv" from_work_dir="outputs/concat_protein.tsv">
+            <filter>not workflow_only and 'concatenated_outputs' in output_options</filter>
+        </data>
+
+        <!-- PTM-Shepherd Results -->
+        <data name="global_profile_tsv" format="tabular" label="${tool.name} on ${on_string}: global.profile.tsv" from_work_dir="outputs/global.profile.tsv">
+               <filter>(not workflow_only) and ('Open' in  wf['workflow_name'])</filter>
+        </data>
+        <data name="global_summary_tsv" format="tabular" label="${tool.name} on ${on_string}: global.modsummary.tsv" from_work_dir="outputs/global.modsummary.tsv">
+               <filter>(not workflow_only) and ('Open' in  wf['workflow_name'])</filter>
+        </data>
+        <data name="global_diagmine_tsv" format="tabular" label="${tool.name} on ${on_string}: global.diagmine.tsv" from_work_dir="outputs/global.diagmine.tsv">
+               <filter>(not workflow_only) and ('Open' in  wf['workflow_name'])</filter>
+        </data>
+
+        <!--
+            TMT/iTRAW Results
+            Only produced by TMT, TRAQ, and SLC workflows
+        -->
+        <collection name="tmt_results" type="list" label="${tool.name} - ${on_string}: TMT Results" >
+            <discover_datasets pattern="__designation__" ext="tabular" directory="outputs/tmt-report"/>
+            <filter>(not workflow_only) and ('TMT' in wf['workflow_name'])</filter>
+        </collection>
+    </outputs>
+
+    <tests>
+      <test expect_num_outputs="11">
+        <param name="inputs" value="default/test1.mzML,default/test2.mzML" ftype="mzml"/>
+        <param name="database_name" value="default/test.fasta" ftype="fasta"/>
+        <param name="manifest" value="default/test.manifest" ftype="tabular"/>
+        <param name="workflow_name" value="Default"/>
```

Comment:
Any chance to add a test for other workflow(s)? Or better later?

Reply:
I can definitely follow up with more tests. I picked these two because they had good coverage of the remaining workflows.